examples/OneOpenAir/OneOpenAir.ino

```diff
@@ -382,6 +382,12 @@ static void createMqttTask(void) {
 }
 
 static void initMqtt(void) {
+  String mqttUri = configuration.getMqttBrokerUri();
+  if (mqttUri.isEmpty()) {
+    Serial.println("MQTT is not configured, skip initialize mqtt client");
```

Comment:
There are log message in `mqttClient.begin`. Just update the log message inside is good enough. Cause MQTT also usage on ESP8266 device too. 

https://github.com/samuelbles07/arduino/blob/bf1a55793768ff8240201e70ac822b6bcd6f022d/src/MqttClient.cpp#L20-L28

Reply:
Yes, but it return false. Hence in the implementation side will print `Connect to MQTT broker failed`. Just realized it is used for others too, will update about this.

Reply:
Don't forget to test with ESP8266 too.